installer/win/WixSharpInstaller/Symphony.cs

```diff
@@ -408,10 +408,25 @@ static string ReplaceProperty( string data, string name, string value )
         // Using regular expressions to replace the existing value in the config file with the
         // one from the property. This is the same as the regex we used to have in the old
         // Advanced Installer, which looked like this: "url"\s*:\s*".*" => "url": "[POD_URL]"
-        return System.Text.RegularExpressions.Regex.Replace(data, @"""" + name + @"""\s*:\s*"".*""",
+        return System.Text.RegularExpressions.Regex.Replace(data, @"""" + name + @"""\s*:\s*"".[^,]*""",
```

Comment:
Maybe I'm missing something, but why do we need this in the old ReplaceProperty?

Reply:
I might be missing something as well, here are my observations.

The behaviour that I noticed is the following one for boolean's function:

first regex version ( @"""" + name + @"""\s*:\s*.*" ) was able to match the entire line, comma included: 
```"ALWAYS_ON_TOP": true,```

as a consequence, the replace method was changing the value of this property and was removing the comma:
```"ALWAYS_ON_TOP": false```. making JSON file not valid.

This is why I added  ```[^,]```` and did the same change for ```ReplaceProperty``` function.

Reply:
Ah ok, I just thought it looked a bit strange with the comma inside the quotes, like it would match "name":"value," ?

Reply:
Got your point. Indeed, it would catch the comma inside quotes. I'm reverting.